Punctuation fixes (#12)
Daeridanii1/successors: data/successor side missions.txt

	deadline 20
	on offer
		conversation

> 			`While in the spaceport on <origin>, you notice a pair of tired-looking Successors lugging around a cart of a curious folded material, taking turns dragging their apparently heavy luggage over the floor. Both perk up substantially when they see you and your ship, however, and pull their equipment towards you with renewed vigor, finally getting close enough to speak.`

Author: Your choice on this change from `planet` to `origin`, but I think this makes it look right, whereas `planet` would make it be “The spaceport on Khosa-Kella-Oa, you notice a pair of…
I may be wrong about this one too, but this does work.

> 			`	While in the spaceport on <planet>, you notice a pair of tired-looking Successors lugging around a cart of a curious folded material, taking turns dragging their apparently heavy luggage over the floor. Both perk up substantially when they see you and your ship, however, and pull their equipment towards you with renewed vigor, finally getting close enough to speak.`

Author: I think it’s supposed to be this way?

```suggestion
			`	While in the spaceport on <origin>, you notice a pair of tired-looking Successors lugging around a cart of a curious folded material, taking turns dragging their apparently heavy luggage over the floor. Both perk up substantially when they see you and your ship, however, and pull their equipment towards you with renewed vigor, finally getting close enough to speak.`
```

		random < 15
	on offer
		conversation

> 			`	Shortly after you disembark the <ship> for the spaceport on <source>, you hear a quiet but noticeable clattering from behind you where your ship is being refueled.`

Author: Maybe do it this way?

```suggestion
			`	Shortly after you disembark the <ship> for the spaceport on <source>, you hear a quiet but noticeable clattering from behind you near where your ship is being refueled.`
```

> 			`Shortly after you disembark the <ship> for the spaceport on <source>, you hear a quiet but noticeable clattering from behind you near where your ship is being refueled.`

Author: The near doesn’t have to be put on, but I think that makes it read better.

					goto fooled

			label what

> 			`	The three Successors are startled at the sound of your voice, their arms whipping around as they turn to face you. One makes a move to run away, but one of the others clamps onto their arm and reels them back alongside them. All three slowly turn a shade of dull purple and their skin seems to contract over the underlying musculature. One of them eventually replies. "We ... ah ... we're part of the repair crew! We've orders to apply this ... hull solution to repair a spot of damage here."`

Author: Then it could be either of these, but the `d` should definitely be added. 

```suggestion
			`	The three Successors, startled at the sound of your voice, whip around and turn to face you. One makes a move to run away, but one of the others clamps onto their arm and reels them back alongside them. All three slowly turn a shade of dull purple and their skin seems to contract over the underlying musculature. One of them eventually replies. "We ... ah ... we're part of the repair crew! We've orders to apply this ... hull solution to repair a spot of damage here."`
```

Reviewer: I still prefer the original wording of "startle" over "are startled" here---_startle_ can be used as a verb, and I prefer the active voice here, especially considering that catching them in the act is meant to be an active moment. Specifying "their arms" is an improvement, though.

			`	"Oh, that's for ... ah ... safety. Yes, 'tis very dangerous to touch if thou'r'nt trained, so 'tis bright in color as a warning." None of the three are wearing any sort of protective equipment that you can see, and by now most of the group have changed color to what you might characterize as a mirthful shade of pink. This is something which one or two are clearly trying to suppress, their skin shifting in gradients in an effort to maintain their previous dull purple hue. One of the others is trying to hide a bag filled with more cans of "hull solution" behind their arms.`
			choice
				`	"Well, if you're working on behalf of the port, then surely I can ask you to remove it as well."`
					goto called
				`	"Nice try, but you and I both know what's going on here."`
					goto called

> 				`	"I think my ship's hull has been well-enough repaired. It might be best for you to attend to whatever other vessels are on your list."`

Author: Not sure which one works here between well and enough, but I think a space is better.

(I had changed it to a space)
